core/src/main/java/org/wso2/testgrid/core/command/FinalizeRunTestplan.java

```diff
@@ -45,19 +62,63 @@
             required = true)
     private String productName = "";
 
+    @Option(name = "--file",
+            usage = "Provide the path to Testgrid configuration file.",
+            aliases = { "-f" })
+    private String jobConfigFile = "";
+
+    @Option(name = "--buildNumber",
+            usage = "Jenkins Build Number",
+            aliases = { "-bn" })
+    private int buildNo;
```

Comment:
We don't need the infrastructure combination? Testgrid build no. is created per infrastructure combination I believe. If this is the Jenkins build number, then, we should explicitly update the 'option' name. This API is a bit confusing IMO. :)

Reply:
Changed this to read the test-plan yaml file as an option to finalize-run-testplan command and read test plan id for updating the status.